index.js

```diff
@@ -83,26 +84,40 @@ const deleteLicense = () => {
   return fs.remove(path.join(basedir, 'LICENSE'))
 }
 
+function setImageNames(composeConfig, keys, name) {
+  keys.forEach(function(key) {
+    composeConfig.services[key].image = `${name}_${key}:latest`
+  })
+}
+
 const updateDockerCompose = async (answers, dbPassword) => {
   const dockerComposePath = path.join(basedir, 'docker-compose.yml')
-  const dokerComposeSource = await fs.readFile(dockerComposePath, 'utf8')
-  const dockerCompose = yaml.parse(dokerComposeSource)
+  const dockerComposeSource = await fs.readFile(dockerComposePath, 'utf8')
+  const dockerCompose = yaml.parse(dockerComposeSource)
   const dbPort = databases[answers.databaseEngine].port
   const db = dockerCompose.services.db
+
+  setImageNames(
+    dockerCompose,
+    ['api', 'frontend', 'redis'],
+    dashify(answers.projectName)
+  )
+
   db.image = answers.databaseEngine
   db.ports = [`${dbPort}:${dbPort}`]
+  const db_user = toSnakeCase(answers.projectName)
```

Comment:
This variable still uses snake case.

Reply:
oops, didn't catch it... sorry
I'll fix it now